pkg/client/connection.go

```diff
@@ -20,8 +20,18 @@ import (
 )
 
 type ApiConnectionDetails struct {
-	ArmadaUrl                   string
-	ArmadaRestUrl               string
+	ArmadaUrl     string
+	ArmadaRestUrl string
+	// After a duration of this time, if the client doesn't see any activity it
+	// pings the server to see if the transport is still alive.
+	// If set below 10s, a minimum value of 10s is used instead.
```

Comment:
Is the `max(time, 10s)` logic implemented?  I can't see it here

Reply:
It's implemented inside the gRPC keepalive logic.